### DISCUSSION TITLE
Resolved failed tests

### DISCUSSION THREAD

**Discussion by BrianBaggs** [test-results-reporter/reporter#253] [Ideas]
Hello,

This is just a suggestion, but I noticed that when I had a couple outdated tests and they got marked as failed, when I went in and clicked the failure analysis and marked them all as not a defect, the test run still has the red X like it was a failed run. It would be nice to at least have the option to mark it as passed, since all of the tests passed except a couple outdated tests that needed to be fixed. 

Thanks!

**Comment by ASaiAnudeep** [comment]
Thank you for your suggestion! We'll explore ways to make the process more intuitive. I will keep this discussion open.

**Reply by BrianBaggs** [reply]
Thank you. This would be a very helpful feature since I rarely have all tests passed, when dealing with 1500 - 2000 tests, sometimes one or two get outdated and then it causes the entire test run to appear as failed, even though it was just an outdated test.

**Reply by ASaiAnudeep** [reply]
I do deal with similar problems. We will try to find the best possible solution. If you have any other approaches to tackle this problem, please add them in this thread.

**Reply by BrianBaggs** [reply]
Sounds good... even if marking the defects as not a defect in failure analysis caused it to go to passed, if all failed tests are marked as not a defect, that would be helpful too. By the way, my name started showing up on the test run so whatever you guys did there, thank you! I am no longer Unknown!
